com.netxms.Client.yaml

```diff
@@ -0,0 +1,47 @@
+app-id: com.netxms.Client
```

Comment:
nextms.com redirects to https://app.nextms.com/ which says service is down?

Reply:
You misspelt the name it NET XMS not NEXT MS :)

Reply:
😅 thanks